src/sfizz/ADSREnvelope.cpp

```diff
@@ -35,7 +35,7 @@ Float ADSREnvelope::secondsToExpRate(Float timeInSeconds) const noexcept
         return Float(0.0);
 
     timeInSeconds = std::max(Float(25e-3), timeInSeconds);
-    return std::exp(Float(-9.0) / (timeInSeconds * sampleRate));
+    return 1.0 / (sampleRate * timeInSeconds);
```

Comment:
I'll deal with this part after the cutoff but what was the reason to change this formula?

Reply:
It's part of the process for being able to customize the EG shapes. I realize it means overwriting what you originally had for this formulat, and apologies for that. In retrospect, with the time, I could've put in the effort to see about reverting to that if the EG shape in unspecified.